### ISSUE TITLE
Design for new message types on groups proposal create form

### ISSUE THREAD

**Issue by erikalogie** [regen-network/groups-ui#30]
If the admin of a group is a group policy, we need to add support for editing groups via proposals.

Required:
- MsgUpdateGroupPolicyDecisionPolicy
- MsgUpdateGroupMembers

Nice to haves:
- MsgUpdateGroupMetadata

Figma files:
https://www.figma.com/file/Mqvg4sP0Gwk1Jn4svsmhDp/Groups-UI?node-id=4800%3A24604&t=hTVHnRRuJHK94ws1-1

**Comment by erikalogie** [comment]
@haveanicedavid

**Comment by erikalogie** [comment]
@ryanchristo @haveanicedavid this design task for new message types is done, the dev task is now unblocked.
